gdal-process/main.go

```diff
@@ -114,9 +117,10 @@ func registerGDALDrivers() {
 	if haveHDF5 {
 		C.GDALRegister_HDF5()
 	}
-	if haveJP2OpenJPEG {
```

Comment:
As discussed, this was the temporal solution to compile GSKY without JPEG for dev purposes. But this code section needs to stay as we need JPEG for the JPEG version of Sentinel 2 in future. Instead, we need to fix the build script to have GDAL built with JPEG support.

Reply:
This issue has now been fixed.